src/pages/settings/safenet.tsx

```diff
@@ -41,7 +41,7 @@ const SafenetContent = ({ safenetConfig, safe }: { safenetConfig: SafenetConfigE
   const safenetGuardAddress = safenetConfig.guards[safe.chainId]
   const safenetProcessorAddress = safenetConfig.processors[safe.chainId]
   const isSafenetGuardEnabled = isVersionWithGuards && sameAddress(safe.guard?.value, safenetGuardAddress)
-  const chainSupported = isSupportedChain(Number(safe.chainId), safenetConfig, SafenetChainType.SOURCE)
+  const chainSupported = isSupportedChain(Number(safe.chainId), safenetConfig)
```

Comment:
Nit: now the chains are unified, I would simply inline the function because it only does a simple `.includes` call

Reply:
Good idea. Done in [here](https://github.com/safe-global/safe-wallet-web/pull/4440/commits/6e3aeac6cd96b57e452ae322347cb2b71f8f5cdc)